tests/st/utils/data.py

```diff
@@ -648,7 +647,21 @@
     'spec': {
         'ipv6Support': False,
         'logSeverityScreen': 'Debug',
-        'netlinkTimeoutSecs': 11,
+        'netlinkTimeout': '11s',
+    }
+}
+
+felixconfig_name1_rev3 = {
```

Comment:
Maybe worth a quick comment above this that the large durations are returned on a get with different units - so it doesn't trip up anyone else using this